pgd_search/tests.py

```diff
@@ -857,6 +858,49 @@ def test_saving_image(self):
         response = self.driver.find_element_by_id("button-save").click()
 
 class ViewTest(TestCase):
-    def home_page_noerror(self):
+
+
+    #Method must start with "test"
+    def test_home_page_noerror(self):
         response = self.client.get(reverse('/'))
         self.assertEqual(response.status_code, 200)
+
+
+
+class CheckDumpTest(TestCase) :
+
+
+    fixtures = ['pgd_search.json']
+
+    def test_download_tsv(self):
+
+        search = Search(segmentLength=5)
+
+        # create associated Search_residues
+        data = {}
+        #data['index'] = 5
+        data['residues'] = 5
+        data['occm_-1'] = 0.58
```

Comment:
Not in this current code, but when i was experimenting, by saving the file, looking and comparing, programmatically comparing, i was able to ascertain it was in the correct column. But yeah, I need to show that here as well. Would altering the corresponding values be fine ?